jupyterhub/overlays/prod/kustomization.yaml

```diff
@@ -3,4 +3,7 @@ resources:
 - ../../bases/jupyterhub
 - ../../bases/custom-images
 - ../../bases/custom-profiles
+- ../../bases
 - membership.yaml
+generators:
+        - ./secret-generator.yaml
```

Comment:
remove indents (see the resources field format above it)

Reply:
fixed